client/components/mb-player-creation.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import socketIOClient from 'socket.io-client';
+
+export default class PlayerCreation extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      endpoint: 'http://localhost:3001',
```

Comment:
Same here regarding the endpoint. Actually, if this never changes, you should leave it out of state altogether.